ot/da.py

```diff
@@ -568,36 +581,40 @@ def joint_OT_mapping_kernel(xs, xt, mu=1, eta=0.001, kerneltype='gaussian',
     if log:
         log = {'err': []}
 
-    a, b = unif(ns), unif(nt)
+    a = nx.from_numpy(unif(ns), type_as=xs)
```

Comment:
same here

Reply:
Added in [89addb6](https://github.com/PythonOT/POT/pull/343/commits/89addb6997f5c3df1e3f40625d893adb3ecca06d)